evalml/pipelines/utils.py

```diff
@@ -105,10 +109,8 @@ def _get_preprocessing_components(
         text_columns
     ):
         pp_components.append(Imputer)
-    index_and_unknown_columns = list(
-        X.ww.select(["index", "unknown"], return_schema=True).columns
-    )
-    if len(index_and_unknown_columns) > 0:
+    index_columns = list(X.ww.select(["index"], return_schema=True).columns)
+    if len(index_columns) > 0:
```

Comment:
Woodwork will only ever have one index column, or it will be `None`, so I think you can combine these two lines into just this:
```python
if X.ww.index:
```